pkg/model/firewall.go

```diff
@@ -441,6 +441,6 @@ func generateName(o *awstasks.SecurityGroupRule) string {
 		src = fi.StringValue(o.SecurityGroup.Name)
 	}
 
-	return fmt.Sprintf("%s-%s-%s-%dto%d-%s", src, direction,
+	return fmt.Sprintf("from-%s-%s-%s-%dto%d-%s", src, direction,
```

Comment:
@olemarkus please correct me if I'm wrong, but won't changing the existing resource name for the terraform resources cause all resources to be re-created, effectively causing downtime while the changes are applied?

cc: @rifelpet